README.md

```diff
@@ -187,9 +187,19 @@ qmk_hid via --rgb-effect 38
 
 ## Building from source
 
-Pre-requisites: Rust, libudev
+Pre-requisites:
+
+- [Rust](https://rustup.rs/)
+- libudev
```

Comment:
Nice! You might want to mention that the `libudev` apt is usually available at `libudev-dev` or just say (most systems: `sudo apt install libudev-dev`.

Reply:
I was thinking about that. But then I was running fedora and it's called something else there...

Reply:
Fair point

Reply:
I added it actually now ;)

Reply:
Sweet, I like that you added instructions for our two most popular distros!